deck.gl/examples/scripting/hello-world.html

```diff
@@ -0,0 +1,55 @@
+<html>
+  <head>
+    <script src="https://unpkg.com/deck.gl@8.3.0-alpha.1/dist.min.js"></script>
```

Comment:
We don't require deck to be the latest... I would use the same as in other scripting examples, <script src="https://unpkg.com/deck.gl@^8.1.0/dist.min.js"></script>

Reply:
Changed